crates/store/src/db/models/queries/accounts.rs

```diff
@@ -887,12 +888,13 @@ pub(crate) struct AccountAssetRowInsert {
 impl AccountAssetRowInsert {
     pub(crate) fn new(
         account_id: &AccountId,
-        vault_key: &Word,
+        vault_key: &VaultKey,
```

Comment:
I think in the latest `next` on `miden-base` this is now called `AssetVaultKey`.